src/main/java/org/jenkinsci/plugins/envinject/EnvInjectComputerListener.java

```diff
@@ -53,10 +55,13 @@ private EnvVars getNewMasterEnvironmentVariables(Computer c, FilePath nodePath,
                 globalPropertiesEnvVars.putAll(((EnvironmentVariablesNodeProperty) nodeProperty).getEnvVars());
             }
 
-            if (nodeProperty instanceof EnvInjectNodeProperty) {
+            final Node node = c.getNode();
```

Comment:
@oleg-nenashev think it's a habit of newbie. Simply don't try to reassign this var and the behaviour will be the same.